#constant for tasks
uint8 TASK_GOHOME = 1
uint8 TASK_POSITON_AND_YAW_CONTROL = 2
uint8 TASK_GOHOME_AND_CONFIRM_LANDING = 3
uint8 TASK_TAKEOFF = 4
uint8 TASK_VELOCITY_AND_YAWRATE_CONTROL = 5
uint8 TASK_LAND = 6

#request
uint8 task    # see constants above for possible tasks
JoystickParams joystickCommand  #Provide Position and Velocity control
uint32 velocityControlTimeMs    #Velocity control time
float32 posThresholdInM  #(Meter)
float32 yawThresholdInDeg  #(Degree)
---
#response
bool result

